.github/workflows/release.yml

```diff
@@ -108,9 +109,7 @@ jobs:
         uses: stefanzweifel/git-auto-commit-action@8756aa072ef5b4a080af5dc8fef36c5d586e521d
         with:
           commit_message: "Update weights for release ${{env.NEW_RELEASE_TAG}}"
-          commit_user_name: Frequency CI [bot]
-          commit_user_email: do-not-reply@users.noreply.github.com
-          commit_author: Frequency CI [bot] <do-not-reply@users.noreply.github.com>
+          file_pattern: "pallets/**/*.rs runtime/common/src/weights/*"
```

Comment:
Limit to just the weights, and remove the commit user as it will just use the default bot+trigger person